graphql-schemas/schema.yml

```diff
@@ -18,12 +18,18 @@
   - { name: pushGatewayCluster, type: Cluster_v1 }
   - { name: alertingServices, type: string, isList: true }
   - { name: endpointMonitoringBlackboxExporterModules, type: string, isList: true }
+  - { name: ldap, type: LdapSettings_v1, isRequired: true}
```

Comment:
for the sake of being "open source", i suggest we don't make this required

Reply:
Yes, that is a good point. I'll make the change.